<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SaveBoxWidget</class>
 <widget class="QWidget" name="SaveBoxWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>235</width>
    <height>396</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="2">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="0" rowspan="2" colspan="2">
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="1" column="0" colspan="3">
       <widget class="QLabel" name="data_dir_label">
        <property name="font">
         <font>
          <pointsize>8</pointsize>
         </font>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="5" column="0" colspan="3">
       <widget class="QPushButton" name="save_single_pushButton">
        <property name="text">
         <string>Save Single</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1" colspan="2">
       <widget class="QSpinBox" name="file_number_spinBox">
        <property name="maximumSize">
         <size>
          <width>127</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="minimum">
         <number>0</number>
        </property>
        <property name="maximum">
         <number>100000</number>
        </property>
        <property name="value">
         <number>0</number>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="file_number_label">
        <property name="text">
         <string>File Number:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1" colspan="2">
       <widget class="QLineEdit" name="file_prefix_lineEdit">
        <property name="maximumSize">
         <size>
          <width>127</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>jkam_capture</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0" colspan="3">
       <widget class="QTabWidget" name="tabWidget">
        <property name="currentIndex">
         <number>0</number>
        </property>
        <widget class="QWidget" name="build_dir_tab">
         <attribute name="title">
          <string>Build Data Dir</string>
         </attribute>
         <layout class="QGridLayout" name="gridLayout_4">
          <item row="2" column="0">
           <widget class="QLabel" name="daily_path_label">
            <property name="text">
             <string>Daily Data Path:</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QLineEdit" name="run_name_lineEdit">
            <property name="maximumSize">
             <size>
              <width>70</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>run0</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLabel" name="data_root_value_label">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="run_name_label">
            <property name="text">
             <string>Run Name:</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="data_root_label">
            <property name="text">
             <string>Data Root:</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QLabel" name="daily_path_value_label">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="0" column="0" colspan="2">
           <widget class="QPushButton" name="data_root_pushButton">
            <property name="text">
             <string>Select Data Root</string>
            </property>
           </widget>
          </item>
          <item row="4" column="0">
           <widget class="QLabel" name="imaging_system_label">
            <property name="text">
             <string>Imaging System:</string>
            </property>
           </widget>
          </item>
          <item row="4" column="1">
           <widget class="QLabel" name="imaging_system_value_label">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="select_dir_tab">
         <attribute name="title">
          <string>Select Data Dir</string>
         </attribute>
         <layout class="QGridLayout" name="gridLayout_5">
          <item row="0" column="0">
           <widget class="QPushButton" name="select_data_pushButton">
            <property name="text">
             <string>Select Data Directory</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
      <item row="6" column="0" colspan="3">
       <widget class="QPushButton" name="run_pushButton">
        <property name="text">
         <string>Begin Run</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="file_prefix_label">
        <property name="text">
         <string>File Prefix:</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0" colspan="3">
       <widget class="QLabel" name="file_path_label">
        <property name="font">
         <font>
          <pointsize>8</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Next File Name: jkam_capture_00000</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
 <designerdata>
  <property name="gridDeltaX">
   <number>10</number>
  </property>
  <property name="gridDeltaY">
   <number>10</number>
  </property>
  <property name="gridSnapX">
   <bool>true</bool>
  </property>
  <property name="gridSnapY">
   <bool>true</bool>
  </property>
  <property name="gridVisible">
   <bool>true</bool>
  </property>
 </designerdata>
</ui>
